lib/snyk-monitor-provider.ts

```diff
@@ -0,0 +1,42 @@
+import { ClusterInfo } from "@aws-quickstart/eks-blueprints";
```

Comment:
We're using double quotes for strings in this file and single quotes elsewhere, easy clean-up task.

Reply:
I try to stick to conventions, but the language doesn't enforce this. I wish it did! Cleaned up in [1216370](https://github.com/snyk-partners/snyk-monitor-eks-blueprints-addon/pull/1/commits/12163702c921d260e510fe2938d0d982e0262170).